module/localisation/FieldLocalisation/data/config/webots/FieldLocalisation.yaml

```diff
@@ -27,3 +27,9 @@ min_observations: 100
 
 # Penalty factor for observations being outside map
 outside_map_penalty_factor: 0.9
+
+# Delay before the particle filter starts (allows odometry to settle)
+start_time_delay: 5
+
+# Starting side of the field (LEFT, RIGHT or EITHER)
```

Comment:
Is this left/right when looking towards the opponent goal?

Reply:
Looking towards our own goal, as the current convention is our goal is in the positive x direction in field space.

Reply:
That will definitely need documenting here and where ever relevant
Old convention was that towards opponents goal is forward (positive)
I checked NUbook and we've got the old convention documented on the maths page.